community/sway/etc/skel/.config/sway/config.d/00-user.conf

```diff
@@ -1,3 +1,6 @@
 # add your customizations here
 # 
-# they will be appended to the configuration
\ No newline at end of file
+# they will be appended to the configuration
+
+# follow windows after moving them
+set $focus_after_move true
```

Comment:
@FallenChromium wdyt about adding such "parameters" to the 00-user file directly? if its not configured, the default behaviour applies.

Reply:
I think it's a good idea, but the problem is 00-user.conf exists in skel, so cp -rf will replace it. I think it will be better to define the defaults in /etc/sway/definitions and allow user to reconfigure it elsewhere in case it will be used dynamically, and in case my suggestion on making this PR statically imported as 2 files will be accepted, we will have to do something else, and I've seen this growing painpoint while creating the dynamic theming PR. To put it simply, we need some kind of user folder to include from ~/.config/sway/config (which will not be overwritten by cp -rf) so we could define variables BEFORE including anything. By that I mean theming variable, as an example. 

As a simpler solution (for this commit exactly, it may not work in other circumstances) we can just use ~/.config/sway/definitions, because it is included earlier than modes folder. For something which is initialized in /etc/sway/definitions though it won't work, so we really shouldn't `include` anything (especially dynamically using variables) at /etc/sway/definitions

Reply:
but: we could add a shortcut script that rsyncs the skel files without overwriting stuff in any .d directories. that would be much more portable and would benefit overall UX. wdyt?

Reply:
That I think is a great idea! Can we do the same for ~/.config/sway/definitions?

Reply:
I think at this point it's worth it to open a separate issue on config includes \ overrides \ overall architecture, because it's an important question and addressing it will make our lives easier. For example I still hate the way waybar is configured, I don't want to change it because I know that something cool will be added in a next release and I will be too lazy to port them over to a new file, so I just replace it :D
It really sucks to replace user-made (if they are modified) configs like this.